diff -up texstudio-4.8.9/src/spellerutility.cpp.1~ texstudio-4.8.9/src/spellerutility.cpp
--- texstudio-4.8.9/src/spellerutility.cpp.1~	2025-09-12 17:02:53.000000000 +0200
+++ texstudio-4.8.9/src/spellerutility.cpp	2025-09-13 15:50:23.601596487 +0200
@@ -11,7 +11,7 @@
 
 #include "spellerutility.h"
 #include "smallUsefulFunctions.h"
-#include "JlCompress.h"
+#include <quazip/JlCompress.h>
 
 int SpellerUtility::spellcheckErrorFormat = -1;
 bool SpellerUtility::inlineSpellChecking = true;
diff -up texstudio-4.8.9/src/templatemanager_p.cpp.1~ texstudio-4.8.9/src/templatemanager_p.cpp
--- texstudio-4.8.9/src/templatemanager_p.cpp.1~	2025-09-12 17:02:53.000000000 +0200
+++ texstudio-4.8.9/src/templatemanager_p.cpp	2025-09-13 15:50:23.601818717 +0200
@@ -1,5 +1,5 @@
 #include "templatemanager_p.h"
-#include "JlCompress.h"
+#include <quazip/JlCompress.h>
 #include "smallUsefulFunctions.h"
 #include <QJsonDocument>
 
diff -up texstudio-4.8.9/texstudio.pro.1~ texstudio-4.8.9/texstudio.pro
